### DISCUSSION TITLE
Enabling passwords import

### DISCUSSION THREAD

**Discussion by sh0rtek** [Alex313031/Thorium-Android#32] [Ideas]
Is it possible to enable passwords import in futures relases? Its quite usefull not to copy and paste every password, but autofill with Thorium....Its blazing fast browser for android. Simply BEST!

**Comment by gz83** [comment]
Thanks for the advice

**Comment by vaclavbenes** [comment]
Hi, I have similar question I use KeePassXC on MacOs and it's nice but I am not able to enable it on android version? Is it supported ?
